README.md

```diff
@@ -36,8 +36,10 @@ The action parameters `api_token` and `owner_name` can also be omitted when thei
 - `APPCENTER_APP_NAME` - App name. If there is no app with such name, you will be prompted to create one
 - `APPCENTER_DISTRIBUTE_APK` - Build release path for android build
 - `APPCENTER_DISTRIBUTE_IPA` - Build release path for ios build
-- `APPCENTER_DISTRIBUTE_DSYM` - Path to your symbols file. For iOS provide path to app.dSYM.zip
+- `APPCENTER_DISTRIBUTE_DSYM` - Path to your symbols (app.dSYM.zip) file 
 - `APPCENTER_DISTRIBUTE_UPLOAD_DSYM_ONLY` - Flag to upload only the dSYM file to App Center
+- `APPCENTER_DISTRIBUTE_MAPPING` - Path to your Android mapping.txt file 
```

Comment:
Maybe this should say "Android mapping file" here instead

Reply:
fixed, thanks